relay-server/src/actors/outcome.rs

```diff
@@ -180,6 +183,7 @@ impl Outcome {
         match self {
             Outcome::Filtered(_) | Outcome::FilteredSampling(_) => OutcomeId::FILTERED,
             Outcome::RateLimited(_) => OutcomeId::RATE_LIMITED,
+            Outcome::CardinalityLimited => OutcomeId::RATE_LIMITED,
```

Comment:
Should we introduce a new outcome instead?

Reply:
It's not strictly necessary; we could also differentiate it purely by reason code, which is what you're doing now. While it's not ideal that two variants to the same outcome ID, we already have a similar setup for dynamic sampling.

If you were to introduce a new outcome, please split this PR in two parts, and first bump all libraries to avoid errors in the consumers.

Reply:
I think it'd be interesting to have this data available internally to evaluate how high/low the limits are, and the impact on costs. I don't know if other parts (e.g. consumers?) need to be updated to handle new outcomes.

Reply:
👍 let's keep it like this and in the future we can split this into a separate outcome id